src/els_utils.erl

```diff
@@ -70,17 +70,23 @@ start_epmd() ->
 %% @doc Returns a project-relative file path for a given URI
 -spec project_relative(uri()) -> file:filename().
 project_relative(Uri) ->
-  Path = els_uri:path(Uri),
-  RootDir = els_config:get(root_uri),
-  RootPath = els_uri:path(RootDir),
-  RelativeBin = string:prefix(string:prefix(Path, RootPath), <<"/">>),
+  Path        = els_uri:path(Uri),
+  RootUri     = els_config:get(root_uri),
+  RootPath    = els_uri:path(RootUri),
+  RelativeBin = prefix(prefix(Path, RootPath), <<"/">>),
```

Comment:
Uhm, this will not work on Windows, will it?

Reply:
My previous version had the benefit of using file-system aware split and join.

But a colleague told me Erlang works with `/` as a path separator internally, but then switches it when it hits the file system.  Which won't work if the URL comes from non-Erlang.

Reply:
Maybe we should revert to my original version? We have an internal utility function now anyway, so no real saving.

And at least it intentionally operates on file paths, rather than strings.

Reply:
But aren't URIs file system independent and always expressed using `/`?

Reply:
What about something like:

```
%% @doc Returns a project-relative file path for a given URI
-spec project_relative(uri()) -> file:filename().
project_relative(Uri) ->
  RootUri     = els_config:get(root_uri),
  RootUriSize = byte_size(RootUri),
  <<RootUri:RootUriSize/binary, RelativePath/binary>> = Uri,
  binary_to_list(filename:basename(RelativePath)).
```

Reply:
Except that does not seem to work, let me try again... :)

Reply:
> Maybe we should revert to my original version? We have an internal utility function now anyway, so no real saving.
> 
> And at least it intentionally operates on file paths, rather than strings.

That is a good point 😜. I'm open to using the original version.

Reply:
Btw, the fact this exists is not comforting...

https://github.com/erlang/rebar3/blob/master/src/rebar_file_utils.erl#L429

Reply:
```
  RootUri     = els_config:get(root_uri),
  RootUriSize = byte_size(RootUri),
  <<RootUri:RootUriSize/binary, RelativePath/binary>> = Uri,
  binary_to_list(string:trim(RelativePath, leading, [$/, $\\])).
```